Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -3133,6 +3152,21 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C17I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
+    '2026D110GenOnly' : {
+        'Geom' : 'Extended2026D110',
+        'Beamspot' : 'HLLHC',
```

Comment:
Hi @AdrianoDee 
Just wonder that we will have the right output with this setting of beamspot?

Reply:
Hi Phat, I see this message only now. Isn't it just dummy it here? I mean: the BS is defined at SIM step no? Or we have something already at GEN step?

Reply:
Hi @AdrianoDee 
Thanks for coming back, I completely forgot about my comment.

`process.VtxSmeared` is part of `process.pgen`. So you will get wrong  parameters for vertex smearing. Using `DBrealisticHLLHC` should give you the proper parameter.

Reply:
`process.pgen = cms.Sequence(cms.SequencePlaceholder("randomEngineStateProducer")+process.VertexSmearing+process.GenSmeared+process.GeneInfo+process.genJetMET, process.PPSTransportTask)`
and
`process.VertexSmearing = cms.Sequence(cms.SequencePlaceholder("VtxSmeared"))`

Reply:
Also typo, it should be `BeamSpot`, not `Beamspot`. With typo, it will pick default BS of `'Gen'`

Reply:
Fix in https://github.com/cms-sw/cmssw/pull/45685

Reply:
Great thanks!